detector/aws_elb_invalid_instance.go

```diff
@@ -10,22 +10,20 @@ import (
 
 type AwsELBInvalidInstanceDetector struct {
 	*Detector
-	IssueType  string
-	TargetType string
-	Target     string
-	DeepCheck  bool
-	instances  map[string]bool
+	instances map[string]bool
 }
 
 func (d *Detector) CreateAwsELBInvalidInstanceDetector() *AwsELBInvalidInstanceDetector {
-	return &AwsELBInvalidInstanceDetector{
-		Detector:   d,
-		IssueType:  issue.ERROR,
-		TargetType: "resource",
-		Target:     "aws_elb",
-		DeepCheck:  true,
-		instances:  map[string]bool{},
+	nd := &AwsELBInvalidInstanceDetector{
+		Detector:  d,
+		instances: map[string]bool{},
 	}
+	nd.Name = "aws_elb_invalid_instance"
+	nd.IssueType = issue.ERROR
+	nd.TargetType = "resource"
+	nd.Target = "aws_elb"
```

Comment:
__[gometalinter]__
* [goconst] 3 other occurrence(s) of "aws_elb" found in: aws_elb_duplicate_name.go:23:14 aws_elb_invalid_security_group.go:24:14 aws_elb_invalid_subnet.go:24:14

<!-- Comment by [SideCI](https://sideci.com) -->